### PR TITLE
fix: Fix some code for datapool module

### PR DESCRIPTION
**Pull request by inchori**
- Fix some code for datapool module.
- Modify README for datapool module.
- Add `flags.AddPaginationFlagsToCmd` for Data Certificates.
- Add `GetAllPools` test for `TestInitGenesis`.

### CODE REVIEW

**Review by hansol-medi — APPROVED** [review]
Thanks!

File: x/datapool/README.md [hunk 1]
```diff
@@ -6,15 +6,19 @@ VALIDATOR=$(panacead keys show {your validator} -a)
...
+# TX_FLAG="--gas auto --gas-adjustment 1.3 --chain-id {your chain ID} --yes"
+
 panacead tx gov submit-proposal wasm-store cw721_base.wasm \
 --title "store NFT contract wasm code" \
 --description "store wasm code for x/datapool module" \
 --instantiate-only-address $MODULE_ADDR \
 --run-as $MODULE_ADDR \
 --deposit "10000000000umed" \
---from $VALIDATOR $TX_FLAG -y
+--from {your validator name} $TX_FLAG -y
```
[the remark this answers is not in the document]
Comment (inchori) [reply x/datapool/README.md:21]: I understand your comment. But when we execute transaction, the `--from` flag's argument is the name of account, not address. For example ,`--from inchori` flag is correct, not `--from panacea1xxx...`. So I changed like that.

**Review by gyuguen — APPROVED** [review]
LGTM 👍 
I left one comment.

File: x/datapool/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (inchori) [reply x/datapool/README.md]: Thanks for pointing out. I was confused. 😂

**Pull request merged**